main/ZsensorDS1820.ino

```diff
@@ -102,7 +102,7 @@ void pubOneWire_HADiscovery() {
 #  ifdef ZmqttDiscovery
   // If zmqtt discovery is enabled, create a sensor topic for each DS18b20 sensor found on the bus, using addr as uniqueID
   if (disc) {
-    Log.notice(F("CreateDiscoverySensor - Found %d" CR), ds1820_count);
+    //Log.notice(F("CreateDiscoverySensor - Found %d" CR), ds1820_count);
```

Comment:
Maybe we can remove the line if we want to be consistent with the others removals

Reply:
ok